Resolve segfault in OpenSP::ParsedSystemId::unparse.
Patch taken from Debian.
https://bugs.gentoo.org/show_bug.cgi?id=196230
--- a/lib/ExtendEntityManager.cxx
+++ b/lib/ExtendEntityManager.cxx
@@ -1238,7 +1238,8 @@
 }
 
 StorageObjectSpec::StorageObjectSpec(const StorageObjectSpec& x)
-: codingSystemName(x.codingSystemName),
+: storageManager(x.storageManager),
+  codingSystemName(x.codingSystemName),
   codingSystem(x.codingSystem),
   specId(x.specId),
   baseId(x.baseId),
@@ -1253,6 +1254,7 @@
 StorageObjectSpec& StorageObjectSpec::operator=(const StorageObjectSpec& x)
 {
   if (this != &x) {
+    storageManager = x.storageManager;
     codingSystemName = x.codingSystemName;
     codingSystem = x.codingSystem;
     specId = x.specId;
